src/FamilyHubs.SharedKernel.Razor/Pages/Shared/_Content.cshtml

```diff
@@ -0,0 +1,2 @@
+﻿@model IDictionary<string,Microsoft.AspNetCore.Html.HtmlString>
```

Comment:
question: what's the role of this empty razor file?

Reply:
Oops, left in by accident - deleted.